### ISSUE TITLE
added to Homebridge as Cannot Bridge more than 149 Accessories at Bridge.Accessory.addBridgedAccessory (/usr/lib/hoobsd/node_modules/hap-nodejs/src/lib/Accessory.ts:599:13).

### ISSUE THREAD

**Issue by alsolaim94** [bwp91/homebridge-ewelink#371]
### What issue do you have? Please be as thorough and explicit as possible.

Unable to register more than 149 Accessories

### Details of your setup.
* Do you use (1) Homebridge UI-X (2) Homebridge CLI or (3) HOOBS? 

Hoobs

* Which version of Homebridge/HOOBS do you have?

1.4.1
LATEST 
* Which platform do you run Homebridge/HOOBS on (e.g. Raspberry Pi/Windows/HOOBS Box)? Please also mention your version of Node.js/NPM if known.

HOOBS BOX
LATEST 
* Which version of this plugin (homebridge-ewelink) do you have? Has the issue started since upgrading from a previous version?

8.15.0
Latest 

* Which eWeLink devices do you have that are causing issues? Please include product models if applicable.

SONOFF DW2-Wi-Fi - Wireless Door/Window Sensor

### Please paste any relevant logs below.

```

BridgeeWeLinkWARNING[ريموت البيت] is not supported, do create a GitHub issue with the following info: [174] {"bindInfos":{"alexa":["56b70842-1a14-4e90-b80c-f029462bd228_26ca1996a20e8bd63617ab272d4eeede1d2d8e32"],"gaction":["56b70842-1a14-4e90-b80c-f029462bd228_ewelinkGoogleHome"],"scene":2},"subDevId":"7007af0d","parentid":"10015c1c54","bleAddr":"7007AF0D","outlet":0,"key":2,"count":302,"actionTime":"2022-05-21T22:22:12.714Z"}.
5/22/2022, 7:42:17 AMeWeLink BridgeeWeLink[مطبخ فراس] initialising with options {"inchChannels":"","inUsePowerThreshold":0,"logging":"standard","showAs":"default"}.
5/22/2022, 7:42:18 AMeWeLink BridgeeWeLinkWARNING[مطبخ فراس SW2] could not be added to Homebridge as Cannot Bridge more than 149 Accessories at Bridge.Accessory.addBridgedAccessory (/usr/lib/hoobsd/node_modules/hap-nodejs/src/lib/Accessory.ts:599:13).
5/22/2022, 7:42:18 AMeWeLink BridgeeWeLinkWARNING[مطبخ فراس] could not be initialised as Cannot set properties of undefined (setting 'firmware') at eWeLinkPlatform.initialiseDevice (/var/lib/hoobs/ewelinkbridge/node_modules/homebridge-ewelink/lib/index.js:1205:41).
5/22/2022, 7:42:18 AMeWeLink BridgeeWeLink[تسريح] initialising with options {"inUsePowerThreshold":0,"logging":"standard","showAs":"default"}.
5/22/2022, 7:42:18 AMeWeLink BridgeeWeLink[تسريح] initialised and found locally with IP [10.235.1.170] [1:MINI].
5/22/2022, 7:42:18 AMeWeLink BridgeeWeLink[انارة المكتب] initialising with options {"inUsePowerThreshold":0,"logging":"standard","showAs":"default"}.
5/22/2022, 7:42:18 AMeWeLink BridgeeWeLink[انارة المكتب] initialised and LAN mode unavailable as not discovered/supported [1:MINI].
5/22/2022, 7:42:18 AMeWeLink BridgeeWeLinkWARNING[درر ريموت] is not supported, do create a GitHub issue with the following info: [174] {"bindInfos":{"alexa":["56b70842-1a14-4e90-b80c-f029462bd228_26ca1996a20e8bd63617ab272d4eeede1d2d8e32"],"gaction":["56b70842-1a14-4e90-b80c-f029462bd228_ewelinkGoogleHome"],"scene":3},"subDevId":"7007b120","parentid":"10015c1d5f","bleAddr":"7007B120","outlet":0,"key":2,"count":126,"actionTime":"2022-05-21T21:55:12.631Z"}.


BridgeeWeLinkWARNING[غرفة ابي وامي] could not be added to Homebridge as Cannot Bridge more than 149 Accessories at Bridge.Accessory.addBridgedAccessory (/usr/lib/hoobsd/node_modules/hap-nodejs/src/lib/Accessory.ts:599:13).
5/22/2022, 7:42:50 AMeWeLink BridgeeWeLinkWARNING[غرفة ابي وامي] could not be initialised as Cannot read properties of undefined (reading 'eweDeviceId') at new deviceSensorContact (/var/lib/hoobs/ewelinkbridge/node_modules/homebridge-ewelink/lib/device/sensor-contact.js:22:62).


```

**Comment by bwp91** [comment]
Hi @alsolaim94 

Wow that’s a lot of devices!

Unfortunately this is a homekit limit of 149 devices per bridge. So the plugin nor any other plugin can go over this limit.

Your options could be:

* Transferring some of the devices to a second ewelink account and then running the plugin twice
* Using the config to remove/ignore some of the devices you don’t use

Unfortunately I can’t help further with this one.

I can definitely help with supporting that device which says “isn’t currently supported”… if you’d be happy to help?

Thanks!

**Comment by alsolaim94** [comment]
Hi @bwp91 

Thank you very much for your very useful feedback.
I got my Hoobs to work with all my devices. I have separated my devices into three accounts of EWeLink and created 3 instances eWeLink bridges in my Hoobs. Then I added them to my Apple Home.

However,the disadvantage side is that I am unable to link three accounts of eWeLink to my Alexa, since each skill required only one eWeLink account. Is there anyway I can have two or more accounts of eWeLink in one alexa account?

Thank you in advance :)
